src/app/Main/Body/QueryPage/QueryPageServices/TableServices/calculateRelativeTableData.service.ts

```diff
@@ -1,10 +1,10 @@
-function calculatePercent(part: number, total: number): string {
-    const percent = total === 0 ? 0 : (part / total) * 100;
-    const percentRounded = percent.toFixed(2);
-    return percentRounded;
+function calculateRelative(part: number, total: number): string {
+    const relative = total === 0 ? 0 : (part / total) * 100;
+    const relativeRounded = relative.toFixed(2);
+    return relativeRounded;
 }
 /**
- * @desc Calculate the percentage of an objectList with absolute numbers
+ * @desc Calculate the relativeage of an objectList with absolute numbers

```

Comment:
"Calculate the relative numbers ..." perhaps?

Reply:
Done. Commit "Little Changes"